src/poliastro/bodies_data.txt

```diff
@@ -0,0 +1,51 @@
+﻿#This file contains data to create some of the most important or polular bodies in Poliastro.
+#
+#Sources for the data at the end
+#
+#Variables given:
+#name, Symbol, k[km**3 / s**2], R[km] , Gravity parent, semi-mayor axis [au], eccentricity [no unit], inclination[deg], mean longitude[deg], longitude of perihelion[deg], longitude of ascending node[deg]
+#
+#
+#       Stars or parentless Bodies:
+#
+Sun , ☉      , 132712440041, 695700 , None, None
+#
+#
+#       Planets of the Solar System 
+#
+Mercury, ☿      , 22031.78 , 2439.7 , Sun,  0.38709927, 0.20563593,  7.00497902, 252.25032350,  77.45779628,  48.33076593            
```

Comment:
If following what I said above, we would rather exclude the orbit parameters from here and instead put the radius of the SOI. In the future we might be interested in adding J2 but that is out of scope for this PR.